docs/guides/using-ronin-db/index.md

```diff
@@ -158,16 +158,22 @@ there are pending migrations. You will have to run `ronin-db migrate`.
 
 <div class="level">
   <div class="level-left">
-    <a class="button" href="../using-the-ronin-cli/">
-      Using the Ronin CLI &raquo;
+    <a class="button" href="../using-ronin-repos/">
+      &laquo; Using Ronin Repos
     </a>
   </div>
 
   <div class="level-item">
-    <a class="button" href="../index.html#guides">
+    <a class="button" href="/docs/#guides">
```

Comment:
You should use a more descriptive commit message than "Update Ronin DB". Something like "fixed the broken Guides link in the Ronin DB guide".